pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -48,31 +48,21 @@ func (h *Helper) GetDockerAuth(registry, authType string) (docker.AuthConfigurat
 	}
 	if _, err := os.Stat(dockercfgPath); err != nil {
 		glog.V(3).Infof("Problem accessing %s: %v", dockercfgPath, err)
-		return authCfg, false
+		return docker.AuthConfiguration{}, false
 	}
 	cfg, err := readDockercfg(dockercfgPath)
 	if err != nil {
 		glog.Errorf("Reading %s failed: ", dockercfgPath, err)
-		return authCfg, false
+		return docker.AuthConfiguration{}, false
```

Comment:
I'm still not completely convinced that this method shouldn't return an error for cases like this.  This looks like a failure to fulfill intent.